server/modules/freckle/projects.js

```diff
@@ -60,20 +60,47 @@ export const createProject = async (projectName) => {
   }
 };
 
+/**
+ * @desc Get a user id with their email address on freckle
+ *
+ * @param {string} email - user email address be assigned to a project
+ *
+ * @returns {promise} - Axios response for request to get userId from email
+ */
+const getUserIdByEmail = async (email) => {
+  const url = `${freckleUrl}/users?freckle_token=${freckleToken}`;
+  try {
+    const users = await axios.get(url);
+    const { id } = users.data.filter(user => user.email === email)[0];
+
+    if (id) {
+      return id;
+    }
+    return null;
+  } catch (_) {
+    return null;
+  }
+};
+
 /**
  * @desc Assign a user to a project on freckle
  *
- * @param {number} userId - Id of the user to be assigned to a project
- * @param {array} projectIds - Array of integer projectIds to be assigned to the user.
+ * @param {number} email - email of the user to be assigned to a project
+ * @param {array} projectId - Array of integer projectId to be assigned to the user.
```

Comment:
The JSDoc here needs some modification.
The `email` should be a string.
The `projectId` is now an integer and not more an array.
This function is not more return a `promise` but instead a response `object`.

Reply:
Fixed this myself.